### DISCUSSION TITLE
Hello I am new here but I know a way to unblock stuff

### DISCUSSION THREAD

**Discussion by Ekanath-K** [3kh0/ext-remover#220] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
1. So you just go to chrome://chrome-signin/ and press "ok"
2. Then, type 456789@bsd48.org and press enter
3. After that, you have to press the "Sign-in options"
4. Then, press "Sign-in with Github"
5. Press "Security" on the bottom 
6. On the top type in "google" and press enter
7. Then press "See topic"
8. Then lastly press the link on your right side "www.google.com"

Now you have most of the websites unblocked :)
Ask me if you want a screen recording if you don't understand.

**Comment by ghost** [comment]
This doesn't work. chrome://chrome-signin/ is a blank window

**Reply by Ekanath-K** [reply]
really this is what is supposed to look like.
![Screenshot 2022-11-06 8 07 58 PM](https://user-images.githubusercontent.com/74334316/200224665-d77346eb-2f1d-4ccf-a5cd-6f3c05e117fe.png)

**Reply by Gagnastiak** [reply]
Its just a blank page

**Reply by md4r2** [reply]
works for securly but not firewall, nice one tho

**Reply by synapt0** [reply]
its just a blank black page

**Reply by Ekanath-K** [reply]
hm, I will try to find another one for you guys. Thanks for the feedback though!

**Reply by ghost** [reply]
yea its just a blank page

**Reply by GGboogie** [reply]
Why doesn't this go all the way up?
And where do you type google.com at?
![Screenshot 2022-11-10 1 36 49 PM](https://user-images.githubusercontent.com/97118713/201212036-629e0445-d2cd-4416-9076-deb242a4c3ae.png)

**Reply by Ekanath-K** [reply]
The only bad thing is its not full screen and a small portion is cut off, but
you have to just type "google" in the "Search Github"
then press "See topic"
then lastly press the link on your right side "www.google.com"

**Comment by Fallenmoon8080** [comment]
I get this: 
![image](https://user-images.githubusercontent.com/112455501/200303707-ebdbd05f-66ac-406c-a535-c4303e7b0833.png)

**Reply by 3kh0** [reply]
Same

![image](https://user-images.githubusercontent.com/58097612/200342411-33f17a1a-557a-46e1-a279-931010518762.png)

**Reply by Ekanath-K** [reply]
Oh dang, maybe I will try to find a another one for you guys

**Reply by The-Man-Is-Me** [reply]
a working version of this would be to use mashawest@d11.org instead of 456789@bsd48.org

**Reply by Ekanath-K** [reply]
I guess I just used a random account in my district that is no longer being used

**Reply by ghost** [reply]
I have a geogebra one, Ill post

**Reply by Ekanath-K** [reply]
oh, yea I know that one, but they blocked that extension for me.

**Comment by PickleIsDaBest** [comment]
Then your school disabled adding more accounts to the chromebook.

**Reply by Ekanath-K** [reply]
No, it works for me, you should try a random account from your district and maybe the "Sign-options" will pop up.

**Reply by PickleIsDaBest** [reply]
Then they blocked using account from outside of the organization possibly

**Reply by PickleIsDaBest** [reply]
It works for me, I had no problems.

**Reply by Ekanath-K** [reply]
ahh ok, nice.

**Comment by SomePVZPlayerMatt** [comment]
GOD DAMN!!!!!


![Screenshot 2022-11-09 09 42 44](https://user-images.githubusercontent.com/116836925/200860640-c8b47448-355c-493b-9940-6dd75472c8c5.png)

**Reply by Ekanath-K** [reply]
wow your school is strictttt

**Reply by 3kh0** [reply]
bro your school tells you your IP damn

**Reply by ghost** [reply]
@3kh0 realy crazy

**Reply by SomePVZPlayerMatt** [reply]
my school district uses contentkeeper

**Comment by synapt0** [comment]
YO YO YO IF THIS ISNT WORKING FOR YOU I FOUND ANOTHER METHOD

go to chrome webstore and get droneblocks or geogebra classic and then use their sign in to google option and enter in the email in this post and do the same method as them after that. good luck

**Reply by GGboogie** [reply]
what?

**Reply by synapt0** [reply]
Sorry I kind of rushed it

Just get the app "Droneblocks" from the Chrome Webstore
Go into it and click the three lines in the top right and press "Sign in with Google"
And then do the rest of the steps already posted on the original post.
